root/static/scripts/edit/externalLinks.js

```diff
@@ -240,6 +240,9 @@ export class ExternalLinksEditor
             } else if (isMusicBrainz(link.url)) {
               error = l(`Links to MusicBrainz URLs are not allowed.
                          Did you mean to paste something else?`);
+            } else if (isBlacklisted(link.url)) {
+              error = l(`Links to this website are not allowed
+                         because it is in our site-wide blacklist.`);
```

Comment:
```suggestion
            } else if (isBlacklisted(link.url)) {
              error = l(`Links to this website are not allowed
                         because it is known to host malware.`);
```

Reply:
I was thinking we'd probably want to expand this one eventually also to stuff like sites known to host illegal downloads, for example, which is why I left a wider message here.

Reply:
Smart links are already being blocked, still we provide an explanatory message. It would be nice to do the same here rather than creating a catch-all. You might want not to refer to “blacklist” in code either.

Reply:
Ok, if you want we can do that, and have basically an "isDecoda" section. I don't expect this malware thing is going to extend to any other sites pretty much ever though.

What's the problem with blacklist? We already use whitelist elsewhere.

Reply:
Editors might be interested to know that a lyrics website they used to visit is actually malicious.

Reply:
Ok, made this explicitly about malware, especially since it seems we might want to add this starzik page there too... this might be more common than I expected.